src/onnx/parse_split.cpp

```diff
@@ -55,12 +55,11 @@ struct parse_split : op_parser<parse_split>
         {
             literal s = parser.parse_value(info.attributes.at("split"));
             s.visit([&](auto v) { vec_splits.assign(v.begin(), v.end()); });
-
-            if(std::accumulate(vec_splits.begin(), vec_splits.end(), int64_t(0)) !=
-               static_cast<int64_t>(lens[tuned_axis]))
-            {
-                MIGRAPHX_THROW("PARSE_SPLIT: sum of split attribute unequal to dim size of axis!");
-            }
+        }
+        else if(args.size() == 2)
+        {
+            auto s = args[1]->eval();
```

Comment:
I think it should first check by calling `can_eval()` otherwise it would return invalid argument in case it is not compile time constant.

Reply:
valid. I guess adding it as part of the if check if there's a second argument?